subprojects/plugins/src/integTest/groovy/org/gradle/api/plugins/JavaBasePluginIntegrationTest.groovy

```diff
@@ -52,116 +47,4 @@ class JavaBasePluginIntegrationTest extends AbstractIntegrationSpec implements J
         file("main/build/classes/java/main").assertHasDescendants("Main.class")
         file("tests/build/classes/java/unitTest").assertHasDescendants("Test.class")
     }
-
-    def "can configure source and target Java versions"() {
```

Comment:
💬  Tests from this file moved to `JavaCompileCompatibilityIntegrationTest.groovy`